source/includes/gcp/_instance_groups.md

```diff
@@ -1 +1,225 @@
-### Instance groups
\ No newline at end of file
+### Instance groups
+An instance group is a collection of VM instances that you can manage as a single entity
```

Comment:
It might be worth mentioning load balancing

Reply:
I will add something.

Reply:
Done